packages/pluggableWidgets/slider-web/src/components/__tests__/Slider.spec.tsx

```diff
@@ -0,0 +1,125 @@
+import { createElement } from "react";
+import { render, screen, cleanup, fireEvent } from "@testing-library/react";
+import { Slider, SliderProps } from "../Slider";
+import { mount } from "enzyme";
+
+describe("Slider", () => {
+    afterEach(cleanup);
+
+    const defaultSliderProps = Object.freeze<SliderProps>({
+        min: -100,
+        max: 100,
+        step: 10
+    });
+
+    it("renders horizontal Slider correctly", () => {
+        const { asFragment } = render(<Slider {...defaultSliderProps} />);
+
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("renders vertical Slider correctly", () => {
+        const { asFragment } = render(<Slider {...defaultSliderProps} vertical />);
+
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("contains correct value", () => {
+        render(<Slider {...defaultSliderProps} value={30} />);
+        const handle = screen.getByRole("slider");
+        expect(handle.getAttribute("aria-valuenow")).toBe("30");
+    });
+
+    it("changes value when clicked", () => {
+        const onChange = jest.fn();
+
+        const wrapper = mount(<Slider min={0} max={100} step={10} onChange={onChange} />);
+
+        const sliderRoot = wrapper.find("div.rc-slider").first();
+
+        sliderRoot.getDOMNode().getBoundingClientRect = () =>
```

Comment:
Awesome test, but I'm just wondering whether we can do it without relying on the `div.rc-slider` css selector and relying on browser functions. 🤔

Reply:
This test works only when I use `div.rc-slider`, selector, otherwise this is not works. If you know better way to get correct node with enzyme, I'm happy to use it